Parser/Client/Browser.php

```diff
@@ -584,6 +624,21 @@ public function parse(): ?array
             $engineVersion = $browserFromUserAgent['engine_version'];
         }
 
+        $family  = self::getBrowserFamily((string) $short);
+        $appHash = $this->browserHints->parse();
+
+        if (null !== $appHash && $appHash['name'] !== $name) {
+            $name    = $appHash['name'];
+            $version = '';
+            $short   = '';
```

Comment:
Does that mean that now browsers might be returned that aren't defined in the list above, and such don't have a short code?

Reply:
right, but I can add short names